### PR TITLE
Gsa 6836 validation fixes

### PR DESCRIPTION
**Pull request by PeterDebneyOasys**

### CODE REVIEW

File: gsa/10.2/General/1D members.gwb
Comment (angelahu005) [inline gsa/10.2/General/1D members.gwb:1]: this file still has sync issues.  However, if i delete the elements of member 3 and recreate the elements from members, the problem goes away.

**Review by angelahu005 — CHANGES_REQUESTED** [review]
please update member 3 of 1D members.gwb so that the elements will pass the sync check.

File: gsa/10.2/General/1D members.gwb
Comment (PeterDebneyOasys) [reply gsa/10.2/General/1D members.gwb:1]: @angelahu005 this model has revealed a bug in the creation / synchronization, so whatever we do at the moment, there will be an error report somewhere: https://ovearup.atlassian.net/browse/GSA-7376
Until that is fixed, does it matter which error is produced, or is there a version with a lesser error?

Comment (PeterDebneyOasys) [reply gsa/10.2/General/1D members.gwb:1]: Meanwhile, I have recreated the elements and uploaded the updated model. It gives a sync warning on my copy of 10.2, but is that fixed on a more recent build...?

Comment (angelahu005) [reply gsa/10.2/General/1D members.gwb:1]: There is no sync issues when I open the updated model with 10.2.0.172

**Review by angelahu005 — APPROVED** [review]

**Review by Abishek20Feb — APPROVED** [review]

**Pull request merged**